Libraries/Azure.psm1

```diff
@@ -272,7 +272,7 @@ Function Create-AllResourceGroupDeployments($SetupTypeData, $TestCaseData, $Dist
 		$RGrandomWord = ([System.IO.Path]::GetRandomFileName() -replace '[^a-z]')
 		$RGRandomNumber = Get-Random -Minimum 11111 -Maximum 99999
 
-		$UseManagedDisks = $CurrentTestData.AdditionalHWConfig.DiskType -contains "managed"
+		$UseManagedDisks = $CurrentTestData.AdditionalHWConfig.DiskType -inotcontains "unmanaged"
```

Comment:
I see the change with below xml update. Are you also going to change pipeline default value?

Reply:
now we switch to use managed disk by default, some of pipeline have override using managed disk, we can change it or not, both are okay; for some of pipeline which didn't specify, it is also okay, just let them use managed disk.

Reply:
I prefer keeping managed disk type by default.